host-ext/crouton/background.js

```diff
@@ -404,6 +411,53 @@ function websocketMessage(evt) {
         }
 
         break;
+    case 'N': /* Raise a notification */
+        /* Payload in JSON format, compatible with chrome.extensions specifications */
+        try {
+            var data = JSON.parse(payload);
+
+            if (!data.type)
+                data.type = "basic";
+            if (!data.iconUrl)
+                data.iconUrl = "icon-128.png";
```

Comment:
I wasn't aware, but it already works. For example, you can add:
```
"iconUrl": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJg\
gg==",
```

(see: http://en.wikipedia.org/wiki/Data_URI_scheme)

Added a short example in the help text.